app/controllers/chambas_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ChambasController < ApplicationController
+  def chambas
```

Comment:
```suggestion
  def index
```

Intentemos seguir las conveciones de REST